src/components/Product/ProductItem.tsx

```diff
@@ -40,6 +43,11 @@ const ProductItem: React.FC<ProductItemProps> = ({
   name,
   price,
 }) => {
+  const dispatch = useDispatch();
+
+  const addItemHandler = () => {
+    dispatch(cartActions.addItem({ name, quantity: 1, price, id }));
```

Comment:
for readability I would do: {name, price, id, quantity: 1}

Reply:
Changed